### PR TITLE
Issue #14. Make welcome screen to look pretty

### PR DESCRIPTION
**Pull request by daredtech**
This pull request adds styling to `Home` component to look welcoming.

## Related Issue

Closes Issue #14 

## Acceptance Criteria

The welcome screen (the screen a user sees before they’ve created their list) looks something like the following (or equivalent, use your creativity!):

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓  | :paintbrush: Styles            ||

### Before

Screen Shot 2020-09-17 at 11 19 27 AM

### After

Screen Shot 2020-09-17 at 11 21 21 AM

## Testing Steps / QA Criteria

* From your terminal, run git checkout vh-sd-home-screen.
* Once on checkout vh-sd-home-screen branch, run npm start to load the server and start the browser.
* Visit the component at http://localhost:3000.

### CODE REVIEW

**Review by CodeMeKathy — APPROVED** [review]
Looks great. Thank you for all your efforts

**Review by brewswain — APPROVED** [review]
Oh this is super cute! and the responsiveness is really nice too

File: src/components/Home/Home.component.jsx [hunk 1]
```diff
@@ -4,8 +4,49 @@ import { useHistory } from 'react-router-dom';
...
+    root: {
+      height: '100vh',
+    },
+    image: {
+      backgroundImage: 'url(https://source.unsplash.com/9m2RZvHS_cU)',
+      backgroundRepeat: 'no-repeat',
+      backgroundColor:
+        theme.palette.type === 'light'
+          ? theme.palette.grey[50]
+          : theme.palette.grey[900],
```
Comment (joeylaguna) [inline src/components/Home/Home.component.jsx:27]: Nice ternary use here :D

File: src/components/Home/Home.component.jsx [hunk 2]
```diff
@@ -4,8 +4,49 @@ import { useHistory } from 'react-router-dom';
...
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    avatar: {
+      margin: theme.spacing(1),
+      backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+      width: '100%', // Fix IE 11 issue.
```
Comment (joeylaguna) [inline src/components/Home/Home.component.jsx:42]: Just curious, what was the IE 11 issue?

**Comment by joeylaguna** [comment]
This looks great! I love seeing that you even had a css fix for an IE11 issue. 

Would love to hear how you like using Material UI as well.

Once the merge conflict gets resolve I'll approve 😄

File: src/components/AddItem/AddItem.component.jsx [hunk 3]
```diff
@@ -44,11 +40,8 @@ const AddItem = props => {
   const addNewItemValue = event => {
     event.preventDefault();
 
-    // Clean Input to remove capitalization, punctuation, and spaces
-    const cleanInput = itemName
-      .toLowerCase()
-      .replace(/[^\w\s]|/g, '')
-      .replace(/[\s]/, '');
+    // Clean Input
```
Comment (jendevelops) [inline src/components/AddItem/AddItem.component.jsx]: Is this comment needed?

**Review by jendevelops — APPROVED** [review]
B-E-A-UTIFUL! 

One suggestion that you might consider is to have some sort of loading icon/animation to make sure the image is loaded before you display the homepage. I noticed it took a split second for the background image to appear on my machine on Chrome.

**Pull request merged**